scanner/updater/export.go

```diff
@@ -28,7 +28,7 @@ func Export(ctx context.Context, outputDir string) error {
 		return err
 	}
 	// create output json file
-	outputFile, err := os.Create(filepath.Join(outputDir, "output.json.ztd"))
+	outputFile, err := os.Create(filepath.Join(outputDir, "output.json.zst"))
```

Comment:
I looked up a bit and there's no .ztd suffix for zstd file but there's .zst. I also checked the Clair code:
https://github.com/quay/clair/blob/5cf5fb8dba87b03ca37ed5a60856f5b4c3e72404/cmd/clairctl/export.go#L74C42-L74C45